.travis.yml

```diff
@@ -8,6 +8,20 @@ env:
     - ORACLE_HOME=/u01/app/oracle/product/11.2.0/xe
     - ORACLE_SID=XE
 
+matrix:
+  include:
```

Comment:
Ive been playing with this for a day and should be able to construct something a little prettier for this and the other patch.